addons/portal/static/src/js/portal_sidebar.js

```diff
@@ -28,7 +28,7 @@ var PortalSidebar = publicWidget.Widget.extend({
     _setDelayLabel: function () {
         var $sidebarTimeago = this.$el.find('.o_portal_sidebar_timeago').toArray();
         $sidebarTimeago.forEach((el) => {
-            var dateTime = deserializeDateTime($(el).attr('datetime')),
+            var dateTime = deserializeDateTime($(el).attr('datetime')).startOf('day'),
```

Comment:
We can use this to clean up the `var`.

Reply:
As it's not our code (portal module), I think it's better to only add the fix I need (Laurent's opinion too)